### PR TITLE
Feature/eventbrite service

### PR DESCRIPTION
**Pull request by Wardormeur**

### CODE REVIEW

File: dojos.js [hunk 1]
```diff
@@ -2137,6 +2156,8 @@ module.exports = function (options) {
         client.end();
         _.each(results.rows, function (dojo) {
           dojo.user_invites = purgeInviteEmails(dojo.user_invites);
+          delete dojo.eventbriteToken;
+          delete dojo.eventbriteWhId;
```
Comment (DanielBrierton) [inline dojos.js]: Maybe centralise and call to it instead of writing the same `delete`s every time?

File: lib/entities/dojo/load.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// TODO: separate perm
+// TODO: use generic bootloader
+module.exports = function () {
+  return function (args, cb) {
```
Comment (DanielBrierton) [inline lib/entities/dojo/load.js]: Why does it return a function instead of the export just being the function?

Comment (Wardormeur) [inline lib/entities/dojo/load.js]: It's based upon our skeleton, it allows us to pass a scope to an act

File: dojos.js (hunk 1 above)
Comment (Wardormeur) [inline dojos.js]: "cleanDojo" ?

Comment (DanielBrierton) [inline dojos.js]: 👍

**Pull request merged**